fmriprep/cli/run.py

```diff
@@ -154,7 +154,7 @@ def main():
 
             if sentry_sdk is not None and 'Workflow did not execute cleanly' not in str(e):
                 sentry_sdk.capture_exception(e)
-        config.loggers.workflow.critical('fMRIPrep failed: %s', e)
+        config.loggers.workflow.critical(f'fMRIPrep failed: {e}')
```

Comment:
since logger is optimized to avoid interpolating until it needs to, I tend to prefer `%` substitution over f strings for any log messages

Reply:
:+1: I will proceed to commit/resolve the suggestions and push force.